### PR TITLE
Remove translations automation

### PR DESCRIPTION
**Pull request by orta**
I don't think we're getting enough value from these automations which would allow a few people to maintain their own language versions of the README. It doesn't seem to be worth all the extra CI time it causes IMO.

### CODE REVIEW

File: .github/workflows/CodeOwnersSelfMerge.yml [hunk 1]
```diff
@@ -1,18 +0,0 @@
-name: Codeowners merging
-on:
-  pull_request_target: { types: [opened] }
-  issue_comment: { types: [created] }
-  pull_request_review: { types: [submitted] }
```
Comment (sandersn) [inline .github/workflows/CodeOwnersSelfMerge.yml:5]: So this was running on every PR, not just ones that edit the READMEs?

File: docs/CODEOWNERS [hunk 2]
```diff
@@ -1,8 +0,0 @@
-# See https://github.com/OSS-Docs-Tools/code-owner-self-merge
-
-README.ru.md @demensky @virzak [translation] [README-RU]
-README.cn.md @Kerminate @kunl [translation] [README-CN]
-README.ja.md @MaySoMusician @sasurau4 [translation] [README-JA]
-README.ko.md @Ailrun @cimfalab [translation] [README-KO]
-README.es.md @LuigiBaute @kant [translation] [README-ES]
-README.pt.md @thzoid @cdias900  [translation] [README-PT]
```
Comment (sandersn) [inline docs/CODEOWNERS:8]: Can we leave this in a form that github will still auto-add these people to PRs that touch these files? I didn't know we had any dedicated people lined up to review them.

**Review by sandersn — CHANGES_REQUESTED** [review]
One question and one request

File: .github/workflows/CodeOwnersSelfMerge.yml (hunk 1 above)
Comment (orta) [reply .github/workflows/CodeOwnersSelfMerge.yml:5]: Yes, and every comment

File: docs/CODEOWNERS (hunk 2 above)
Comment (orta) [reply docs/CODEOWNERS:8]: I don't think so, you can't assign someone to a PR if don't have write access

**Pull request merged**